app/core/components/OnboardingQuests.jsx

```diff
@@ -300,11 +308,7 @@ const OnboardingAvatar = ({ data, expire, signature, refetch }) => {
         >
           <div className="flex flex-grow">
             <div className="">
-              <UserAvatar
-                size={32}
-                className="h-5 w-5 text-indigo-400 dark:text-indigo-200"
-                aria-hidden="true"
-              />
+              <UserAvatar size={32} className="h-5 w-5 text-indigo-400 dark:text-indigo-200" />
```

Comment:
Could you please tell me why this is edited? This seems like an arbitrary edit and out of scope for this PR. 

It is good practice to not make edits unrelated to the scope as it is high-risk for creating problems/confusion. 

for what it's worth - `aria-hidden` is used to remove icons (among other things) from screenreaders. This removal needs to be undone because it's necessary here.

Reply:
```suggestion
              <UserAvatar size={32} className="h-5 w-5 text-indigo-400 dark:text-indigo-200" aria-hidden="true" />
```

Reply:
@chartgerink I don't remember actively removing this. Thanks for adding this back.

I 100% agree with not making arbitrary edits! Thanks for catching.

And thanks for explaining what `aria-hidden` does.